doc/reference/ovn-internals.md

```diff
@@ -0,0 +1,168 @@
+# LXD OVN implementation
+
+Open Virtual Networks (OVN) is an open source Software Defined Network (SDN) solution.
+OVN is designed to be incredibly flexible.
+This flexibility comes at the cost of complexity.
+OVN is not prescriptive about how it should be used.
+
+For LXD, the best way to think of OVN is as a toolkit.
+We need to translate networking concepts in LXD to their OVN analogue and instruct OVN directly, at a low level, what to do.
+
+This document outlines LXD's approach to OVN in a basic setup.
+It does not yet cover load-balancers, peering, forwards, zones, or ACLs.
+
+For more detailed documentation on OVN itself, please see:
+
+- [Overview of OVN and SDNs](https://ubuntu.com/blog/data-centre-networking-what-is-ovn).
```

Comment:
Perhaps these links can go in the "relevant links" section?

Reply:
Happy to add them there as well but I'd like to keep them prominent so that they are "required reading", contextually.